pkgs/build-support/release/debian-build.nix

```diff
@@ -87,12 +90,12 @@ vmTools.runInLinuxImage (stdenv.mkDerivation (
         echo "file deb-extra $(ls $i/debs/*.deb | sort | head -1)" >> $out/nix-support/hydra-build-products
```

Comment:
ls is meant to be for human consumption. Not to be used in scripts.

Reply:
Sorry, seems I've missed your message a long time ago. Do you perhaps remember what did you mean? I haven't changed this line so I'm unsure.

Reply:
@abbradar 0xABAB was banned from the project about half a year ago, since they left lots of unnecessary/toxic comments on issues/PRs.